boskos/BUILD.bazel

```diff
@@ -1,5 +1,8 @@
 package(default_visibility = ["//visibility:public"])
 
+load("@io_bazel_rules_docker//go:image.bzl", "go_image")
```

Comment:
Can we use prow_image so this has a good publish date?

Reply:
done.